corehq/blobs/zipdb.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+import base64
+from hashlib import md5
+from os.path import commonprefix, join, sep
+import zipfile
+
+from corehq.blobs import BlobInfo, DEFAULT_BUCKET
+from corehq.blobs.exceptions import BadName
+from corehq.blobs.interface import AbstractBlobDB, SAFENAME
+
+
+class ZipBlobDB(AbstractBlobDB):
+    """Blobs stored in zip file. Used for exporting a domain's blobs
+    """
+
+    def __init__(self, slug, domain):
+        self.zipname = 'export-{domain}-{slug}-blobs.zip'.format(domain=domain, slug=slug)
```

Comment:
What do you think of opening the zip file here in `__init__` and adding a `close` method (called by `BlobDbBackendExporter.processing_complete`) rather than open/close on each `copy_blob`. Seems like it might be useful to check if the export file already exists at the beginning of the export and refuse to modify it. Could add an `--append` flag to the management command if you want to override that sanity check, but I assume that wouldn't be the default case.

Also, do you think the export and import management commands should document in their help that they use this filename (in the current directory)? Might also be nice to add an option to specify this file path on the command line.